src/Data/Array.purs

```diff
@@ -551,6 +553,33 @@ alterAt i f xs = maybe Nothing go (xs !! i)
 -- Transformations -------------------------------------------------------------
 --------------------------------------------------------------------------------
 
+-- | Inserts the given element in-between each element in the array. The array
+-- | must have two or more elements for this operation to take effect.
+-- |
+-- | For example:
+-- | ```
+-- | intersperse " " [ "a", "b" ] == [ "a", " ", "b" ]
+-- |
+-- | intersperse 0 [ 1, 2, 3, 4, 5 ] == [ 1, 0, 2, 0, 3, 0, 4, 0, 5 ]
+-- | ```
+-- | If the array has one or zero elements, the outputted array is the same
+-- | as the input array.
+-- | ```
+-- | intersperse " " [] == []
+-- | intersperse " " ["a"] == ["a"]
+-- | ```
```

Comment:
```suggestion
-- | Inserts the given element in between each element in the array. The array
-- | must have two or more elements for this operation to take effect.
-- |
-- | ```purs
-- | intersperse " " [ "a", "b" ] == [ "a", " ", "b" ]
-- | intersperse 0 [ 1, 2, 3, 4, 5 ] == [ 1, 0, 2, 0, 3, 0, 4, 0, 5 ]
-- | ```
-- |
-- | If the array has one or zero elements, the resulting array is the same
-- | as the input array.
-- |
-- | ```purs
-- | intersperse " " [] == []
-- | intersperse " " ["a"] == ["a"]
-- | ```
```

Reply:
Some minor typo / code block fixes here ^

Reply:
I don't think the `purescript` part after the three opening ticks does anything other than add noise to the docs (i.e.  the word "purescript" appears before the code block or as the first word in it or something).

Reply:
It doesn't do anything in the docs at the moment, though it _could_ add syntax highlighting. The word "purescript" doesn't appear in the code block -- see: https://pursuit.purescript.org/packages/purescript-arrays/5.3.1/docs/Data.Array#v:reverse

Reply:
Ok. I've added it